### ISSUE TITLE
graphql/validate.lua:18: attempt to index field 'fields' (a nil value)

### ISSUE THREAD

**Issue by sharonovd** [tarantool/cartridge#134]
In GitLab by @olegrok on Jul 8, 2019, 10:35

Unhandled error

**Comment by rosik** [comment]
This happens when you perform invalid request and try to index simple value, which is not an object

For example:

```graphql
  {
    cluster {
      self {
        uri { x }
      }
    }
  }
```

**Comment by olegrok** [comment]
Could we show more informative error message to user?

**Comment by rosik** [comment]
Resolved in #248. Now it says `Field "x" is not defined on type "String"`
